en/docs/administer/changing-to-mariadb.md

```diff
@@ -91,181 +170,31 @@ Follow the steps below to change the type of the default datasource.
     </tr>
     <tr class="odd">
     <td><p><strong>testOnBorrow</strong></p></td>
-    <td>The indication of whether objects will be validated before being borrowed from the pool. If the object fails to validate, it will be dropped from the pool, and another attempt will be made to borrow another.</td>
+    <td>Indicates Whether objects will be validated before being borrowed from the pool. If the object fails to 
+    validate, it will be dropped from the pool, and another attempt will be made to borrow another.</td>
     </tr>
     <tr class="even">
-    <td><strong>validationQuery</strong></td>
-    <td>The SQL query that will be used to validate connections from this pool before returning them to the caller.</td>
+    <td><p><strong>defaultAutoCommit</strong></p></td>
+    <td>Indicates Whether to commit database changes automatically or not.</td>
```

Comment:
Please change the W in Whether to simple case.

Reply:
done